src/adapters/manta.ts

```diff
@@ -60,6 +61,22 @@ export const calamariRoutersConfig: Omit<CrossChainRouterConfigs, "from">[] = [
       weightLimit: DEST_WEIGHT,
     },
   },
+  {
+    to: "kusama",
+    token: "KSM",
+    xcm: {
+      fee: { token: "KSM", amount: "11523248" },
+      weightLimit: DEST_WEIGHT,
```

Comment:
An [example tx](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fws.internal.kusama.systems#/explorer/query/0x5f4ea5aebc32f50e24b6ec3b69e879d4fb924f95c80cf204c908011f690d4adc) on our staging testnet(v0.9.34), shows that fee=115232479,and weight=298368000.

While on current Kusama(v0.9.35), the weight=298368000, but the fee=104812992. Because Relaychain/Parachain may rebenchmark, and change weight in different version. I feel like we should use some version mechanism, otherwise if the fee may work on staging but failed on production. Or we can use something like max value.

Reply:
The original DEST_WEIGHT came from polkawallet contributors, so I am hesitant to change it to be lower. Other projects are already using this configuration in production, so we can't change it on them. I think polkawallet contributors set DEST_WEIGHT to something very high in case things get re-benchmarked to be more expensive. Is there anything wrong with keeping it too high for now?

Reply:
set weight too high has no problem. but the fee here is too low.

Reply:
Sure, sounds good. I think `115232479` vs `11523248` is a rounding mistake / typo.

Reply:
I think I will just use the correct production fee, since it is more important for this to be accurate in production than for the testnet. This library doesn't really conveniently support testnets yet.